### PR TITLE
Don't pollute output when testing Postgres connection

### PR DESCRIPTION
**Pull request by aaa2000**
Following issue #31 and the comment https://github.com/api-platform/demo/pull/31#issuecomment-422172608

### CODE REVIEW

**Comment by aaa2000** [comment]
I restore the command `pg_isready` removed from #31 and use the same DATABASE_URL of https://github.com/api-platform/api-platform but there is the same error `/tmp:5432 - no response` that https://github.com/api-platform/demo/issues/30#issuecomment-418642914 

I have the postgres image 
```
postgres                                        9.6-alpine          5433943ddb9f        7 days ago          38.6MB
```

File: api/docker/php/docker-entrypoint.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'bin/console' ]; then
 	if [ "$APP_ENV" != 'prod' ]; then
 		composer install --prefer-dist --no-progress --no-suggest --no-interaction
 		>&2 echo "Waiting for Postgres to be ready..."
-		until bin/console doctrine:query:sql "SELECT 1" --quiet; do
+		until pg_isready --timeout=0 --dbname="${DATABASE_URL}"; do
```
Comment (vincentchalamon) [inline api/docker/php/docker-entrypoint.sh]: Even if we prefer using a Postgres database for performances reasons, this command definitively connects this entrypoint to the database system & driver. I don't find it very useful, especially if the final user wants to use a different system.
WDYT @dunglas?

Comment (teohhanhui) [reply api/docker/php/docker-entrypoint.sh]: There is a better way. Thanks to the Sylius peeps who used it: https://github.com/Sylius/Sylius-Standard/blob/943125142f0267ac49903efeaab85e24de08424d/docker/php/docker-entrypoint.sh#L20-L23

Let's adopt the same method as well.

Comment (vincentchalamon) [reply api/docker/php/docker-entrypoint.sh]: It seems the same, but what is the purpose of using ` >/dev/null 2>&1` instead of `--quiet`?

Comment (dunglas) [reply api/docker/php/docker-entrypoint.sh]: We used this Doctrine command long ago, but I can't remember why we switched. `pg_isready` looks more reliable anyway.

> Even if we prefer using a Postgres database for performances reasons, this command definitively connects this entrypoint to the database system & driver. I don't find it very useful, especially if the final user wants to use a different system.

It's the same for the PDO driver and so on. The entrypoint is part of the infrastructure, to me it's ok and intended to couple it with the services we use.

Comment (dunglas) [reply api/docker/php/docker-entrypoint.sh]: Anyway, please update `api-platform/api-platform` first, and the demo after by copying it.

**Comment by vincentchalamon** [comment]
Is this PR still maintained?

**Comment by soyuka** [comment]
We should be consistent with https://github.com/api-platform/api-platform/blob/master/api/docker/php/docker-entrypoint.sh#L25 closing this

**Pull request closed without merging**